packages/flame/lib/src/text/styles/flame_text_style.dart

```diff
@@ -15,16 +15,20 @@ import 'package:flame/text.dart';
 abstract class FlameTextStyle {
   const FlameTextStyle();
 
+  /// Creates a new [FlameTextStyle], preferring the properties of [other]
+  /// if present, falling back to the properties of `this`.
   FlameTextStyle copyWith(covariant FlameTextStyle other);
 
+  /// Merges two [FlameTextStyle]s together, preferring the properties of
+  /// [style1] if present, falling back to the properties of [style2].
   static T? merge<T extends FlameTextStyle>(T? style1, T? style2) {
     if (style1 == null) {
       return style2;
     } else if (style2 == null) {
       return style1;
     } else {
       assert(style1.runtimeType == style2.runtimeType);
-      return style1.copyWith(style2) as T;
+      return style2.copyWith(style1) as T;
```

Comment:
Doesn't it make more sense in the order it was in before? I think that is what I would expect from a merge function, that the first argument is the base and you overwrite its properties with the second one, especially if this would be a merge function taking in more than two arguments.

Reply:
Similarly to this for example: https://api.flutter.dev/flutter/package-collection_collection/mergeMaps.html

Reply:
Good point; I had chosen this particular order because it makes the copyWith neater:
![image](https://github.com/flame-engine/flame/assets/882703/da599388-ea6b-43b1-bd08-c68184578ad5)
But I think your argument is stronger, so I will swap